Mathlib/Topology/Spectral/Prespectral.lean

```diff
@@ -0,0 +1,104 @@
+/-
+Copyright (c) 2025 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import Mathlib.Order.Ideal
+import Mathlib.Topology.Sets.Compacts
+import Mathlib.Topology.Sets.OpenCover
+import Mathlib.Topology.Spectral.Hom
+
+/-!
+
+# Prespectral spaces
+
+In this file, we define prespectral spaces as spaces whose lattice of compact opens forms a basis.
+
+-/
+
+open TopologicalSpace Topology
+
+variable {X Y : Type*} [TopologicalSpace X] [TopologicalSpace Y]
+
+/-- A space is prespectral if the lattice of compact opens forms a basis. -/
+@[mk_iff]
```

Comment:
stacks tag?

Reply:
Prespectral isn't something defined elsewhere so there isn't a corresponding tag from stacks.

Reply:
I would have put the Stacks tag for spectral spaces, and a comment saying we are not assuming sober and blah